instance/static/js/src/instance.js

```diff
@@ -75,9 +75,18 @@ app.controller("Index", ['$scope', 'Restangular', 'OpenCraftAPI', '$q',
             });
         };
 
-        $scope.select = function(selection_type, value) {
-            $scope.selected[selection_type] = value;
-            console.log('Selected ' + selection_type + ':', value);
+        $scope.select = function(instance) {
+            $scope.loading = true; // Display loading message
```

Comment:
Nit: I'd move this into the anonymous function below, because that's the function the clean-up is relating to.